pom.xml

```diff
@@ -658,6 +658,13 @@
         <version>${version.org.json}</version>
       </dependency>
 
+      <dependency>
+        <groupId>org.hsqldb</groupId>
```

Comment:
The same question - what is the advantage of using explicit Database to Wildfly/EAP's default one?